tools/automation/tests/__init__.py

```diff
@@ -42,6 +102,20 @@ def execute(args):
         output('Running in CI Mode')
         selected_modules = [('All modules', 'azure.cli', 'azure.cli'),
                             ('CLI Linter', 'automation.cli_linter', 'automation.cli_linter')]
+
+        modified_modules = _extract_top_level_modified_modules()
+        if any(base_mod in modified_modules for base_mod in ['core', 'testsdk', 'telemetry']):
+            pass
+        else:
+            test_paths = []
+            for mod in modified_modules:
+                try:
+                    test_paths.append(os.path.normpath(test_index[mod]))
```

Comment:
>test_paths [](start = 20, length = 10)

what if there is duplicated test path? how about use set to have unique value and then transfer to list?

Reply:
test_path won't be duplicated becase it's like `c:\\users\\jianzen\\documents\\repo\\azure-cli\\src\\azure-cli\\azure\\cli\\command_modules\\storage\\tests\\latest\\test_storage_acl_scenarios.py:StorageAccessControlListTests.test_storage_container_acl_scenarios` .
It's like command_module_folder + tests + profile_name + test_file_name + test_class_name + test_case_name.

Reply:
Changed to set()